src/components/SidebarError/SidebarError.scss

```diff
@@ -0,0 +1,77 @@
+.sa-sidebar-error {
+  color: var(--sa-neutral-color-1);
+
+  h1 {
+    @include typographyMediumL;
+  }
+
+  .sidebar-error-view-wrapper {
+    height: 318px;
+    text-align: center;
+    background: var(--sa-neutral-color-2);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 4px;
+
+    .sidebar-error-view {
+      img {
+        margin-bottom: 16px;
+        vertical-align: top;
+      }
+
+      h3 {
+        @include typographyMediumXs;
+        color: var(--sa-error-color-1);
+        text-align: center;
+      }
+    }
+  }
+
+  .sidebar-error-paper-input {
+    padding: 16px;
+    text-align: center;
```

Comment:
this text-align: center; and align-items: center; can also be removed, instead add just text-align: center; to .sidebar-error-button